pkgs/development/compilers/gmqcc/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, pkgs, ... }:
+pkgs.stdenv.mkDerivation rec {
+    pname = "gmqcc";
+    version = "297eab9";
+
+    src = pkgs.fetchgit {
+      url = "https://github.com/graphitemaster/gmqcc";
+      rev = "297eab9e5e2c9cc4f41201b68821593a5cf9a725";
+      sha256 = "1hl2qn7402ia03kjkblj4q4wfypxkil99sivsyk2vrnwwpdp4nzx";
+    };
+    #sourceRoot = "${pname}-${version}/Quake";
+
+    makefile = "Makefile";
+
+    installPhase = ''
+      runHook preInstall
+      mkdir -p $out/bin
+      cp gmqcc $out/bin
```

Comment:
please use `install` instead of `cp` to copy and apply the correct permissions

Reply:
@lucasew do I need to set any _specific_ permissions for this?